slamd/common/session_controller.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from flask import Blueprint, make_response, request, redirect
+
+from slamd.common.error_handling import SlamdUnprocessableEntityException, ValueNotSupportedException
+from slamd.common.session_service import SessionService
+
+session_bp = Blueprint('session', __name__, url_prefix='/session')
```

Comment:
I had trouble naming this one, since "session" collided with the flask session name. I renamed it to "session_blueprint", but I'm open to suggestions for a better name.